frontend/src/mocks-server/__mockdata__/logs_query_range.ts

```diff
@@ -0,0 +1,45 @@
+export const logsQueryRangeSuccessResponse = {
+	status: 'success',
+	data: {
+		resultType: '',
+		result: [
+			{
+				queryName: 'A',
+				series: null,
+				list: [
+					{
+						timestamp: '2024-02-15T21:20:22Z',
+						data: {
+							attributes_bool: {},
+							attributes_float64: {},
+							attributes_int64: {},
+							attributes_string: {
+								container_id: 'debian',
+								container_name: 'hotrod',
+								driver: 'T703638C',
+								eta: '2m0s',
+								location: 'frontend/best_eta.go:106',
+								log_level: 'INFO',
+								message: 'Dispatch successful',
+								service: 'frontend',
+								span_id: '0e727d00a1560dc7',
```

Comment:
Please use the dummy data, instead of real one.

Reply:
done